### PR TITLE
Event prediction - request view (branch: Ui organizers)

### LINKED ISSUE #55: "Cancel your event" page 1 UI
https://www.figma.com/file/4FTT6neJVlMD5CQHTTpprF/Covid?node-id=0%3A1

### PR DESCRIPTION
**Pull request by Mati-Roy**
Closes #55 
Addresses part of #56, but not fully
### Added
- Template for `request-event-evaluation` (label "Event Prediction")
- Template for sidebar (unused ?!?)
### Changed
- Template for `model` (label "Request model")

### CODE REVIEW

**Comment by Bachibouzouk** [comment]
@hnykda - I can make the merge of #62 and solve the potential conflicts of #55

File: src/server/main.py [hunk 1]
```diff
@@ -35,10 +37,19 @@
     return templates.TemplateResponse("model.html", {"request": request},)
 
 
-@app.get("/selections")
+@app.get("/request-event-evaluation")
```
Comment (Bachibouzouk) [inline src/server/main.py]: @hnykda - was this supposed to be replaced?

**Comment by Bachibouzouk** [comment]
@hnykda - Can you look at the page if it correspond to what was wanted? I don't find the page responsive with the map, the sidebar (with fondation I had it responsive...) is pushed below the map. If it is good, let's merge and improve more later

File: src/server/main.py (hunk 1 above)
Comment (hnykda) [reply src/server/main.py]: Yes, that's fine.

File: src/server/templates/request-event-evaluation.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  </div>
+  <hr />
+  <br />
+  <form action="/result-event-evaluation">
+    <div class="form-group">
+      <div class="row">
+        <div class="col-lg-5 col-sm-12">
+          <label for="location">Where is your event?</label>
+
+          <select class="form-control" id="exampleFormControlSelect1">
```
Comment (hnykda) [inline src/server/templates/request-event-evaluation.html]: This will need data from us.

File: src/server/templates/model.html [hunk 3]
```diff
@@ -3,13 +3,133 @@
...
-  <div>
-    <!-- Initialize a select button -->
-    <select id="selectButton"></select>
+  <br />
+  <div class="row">
+    <div class="col-lg-5 col-sm-12">
+      <select
+        id="selectButton"
+        class="form-control large"
+        id="exampleFormControlSelect1"
```
Comment (hnykda) [inline src/server/templates/model.html]: This is populated automatically from the lineplot.js

**Pull request merged**